__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../db/connection");
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const { describe, test, expect } = require("@jest/globals");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET 200: should return an array of topics objects ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body.topic.length).toBeGreaterThan(0);
+        expect(Array.isArray([response])).toBe(true);
+        response.body.topic.forEach((topic) => {
+          console.log(topic);
+          expect(topic).toHaveProperty("description");
+          expect(topic).toHaveProperty("slug");
+        });
```

Comment:
Great check here, a few notes:
1. Remember we're returning multiple topics here so the key the data comes back on should be `topics`
2. you've checked the properties are there but you can and should also check the properties contain the right data types.